docs/site/sidebars/lb4_sidebar.yml

```diff
@@ -880,3 +889,4 @@ children:
   - title: 'Considerations for GDPR readiness'
     url: Deploy-for-GDPR-readiness.html
     output: 'web, pdf'
+    
```

Comment:
there's a warning icon. is it because there's an extra tab? (I encountered this before :) )